src/views/migration/components/MigrationToolPhrase.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { Trans } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import ExternalLink from '@kubevirt-utils/components/ExternalLink/ExternalLink';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { Alert, AlertVariant } from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import {
+  MIGRATION_TOOL_DOCUMENTATION_URL,
+  PackageManifestKind,
+  SubscriptionsKind,
+} from '../constant';
+
+type MigrationToolPhraseProps = {
+  operator: PackageManifestKind;
+  subscription: SubscriptionsKind;
+};
+
+const MigrationToolPhrase: React.FC<MigrationToolPhraseProps> = ({ operator, subscription }) => {
+  const { t } = useKubevirtTranslation();
+
+  if (!operator)
+    return (
+      <Alert
+        variant={AlertVariant.danger}
+        isInline
+        title={t(
+          'The Migration Tool operator could not be installed because it is not available on the cluster',
+        )}
+      >
+        <ExternalLink text={t('Learn more')} href={MIGRATION_TOOL_DOCUMENTATION_URL} />
+      </Alert>
+    );
+
+  if (!subscription)
+    return (
+      <Trans t={t} ns="plugin__kubevirt-plugin">
+        The Migration Tool for Virtualization Operator facilitates the migration of multiplce
+        Virtual Machine workloads to Openshift Virtualization. Clicking the install migration tool
+        button will take you to the{' '}
+        <Link to={'/operatorhub'}>
+          OperatorHub. <ExternalLinkAltIcon />
+        </Link>
+      </Trans>
+    );
+
+  return (
+    <>
+      {t(
+        'The Migration Toolkit operator is successfully installed, please deploy Migration Tool instance in order to use the Migration Toolkit',
```

Comment:
see https://github.com/kubevirt-ui/kubevirt-plugin/pull/379/files#r867438065
the installation of MTV is more complex then just installing the MTV instance, we can leave the "how to install" to the official documentation.